modules/develop/pages/data-transforms/build.adoc

```diff
@@ -170,6 +222,29 @@ fn my_transform(event: WriteEvent, writer: &mut RecordWriter) -> anyhow::Result<
   // Allow errors with writes to escape
   return writer.write(record)
 }
+----
+--
+JavaScript::
++
+--
+
+[source,js]
+----
+import { onRecordWritten } from "@redpanda-data/transform-sdk";
+
+// Register your transform function.
+onRecordWritten((event, writer) => {
+  const record = event.record;
+  if (!record.key) {
+    // Handle the error internally by logging it
+    console.error("Error: Record key is nil");
+    // Skip this record and continue to process other records
+    return null
```

Comment:
nit: consistency with `;`

Also you can just `return;` this method returns `void` in typescript